### PR TITLE
Adds Mechanic Locker and personal soft suit

### PR DESCRIPTION
**Pull request by Xantholne**
Adds a Mechanic access only locker to the left of their disposal bin, inside is a special sprite Steam punk themed softsuit for them to wear as suggested on 
https://nanotrasen.se/forum/topic/10602-add-more-access-to-mechanic-job/#comment-87834

7995882b1d7ffd99593a27a4c84d0b5d
Special sprite for every species that is different when wearing softsuits while keeping to the original look of the regular more bulky looking softsuit. Mostly inspired by steam punky diving suits with lots of bronze and gears.

🆑 Xantholne
rscadd: Mechanics now have a personal locker and inside is a special steam punk soft suit
/🆑

### CODE REVIEW

**Comment by GunDOSMk1** [comment]
This is great, mechanics seriously needed a hard suit. My only problem is that, the shade of yellow your using looks a little vomity. Buuuuut that may just be my screen!

**Comment by Aurorablade** [comment]
I really do not like the sprite..and iffy about them having a hardsuit..and i play mechnic more often then not.

**Comment by FlattestGuitar** [comment]
Kinda iffy on the sprites as well, they look way too messy.

:+1: for the PR though. Lately we've had Sec Pilots rushing the Derelict roundstart in order to steal the softsuits there. Without a suit you're pretty useless in space.

**Comment by Anticept** [comment]
Why a hardsuit and not a softsuit?

If anything, they need the gloves and a softsuit more than just a hardsuit!

**Comment by Xantholne** [comment]
Went with hardsuit for compactability of the helmet while also still being Engineery. If they want a soft suit they can pocket then they can go out into space and get one instead. Either way their helmets will have weld protection. 

Glove wise. No. They can either go into space for combats like most or bug the hell out of Engineers for a pair.

**Comment by SamHPurp** [comment]
Like the concept of them getting their own locker and spacesuit... I don't like the sprite, nor that it is a hardsuit, to be honest.

**Comment by SamHPurp** [comment]
Also you didn't map merge. You're going to need to do that. If you need a walk through on it, I'm happy to help.

**Comment by Xantholne** [comment]
If people dislike it being a hardsuit that much I can probably change it to a basic space suit with the weld protection on the helmet like the regular syndies ones which I can't say I'd particularly enjoy since you wouldn't be able to toggle your helmet or not. I don't see it as too much of an issue since you're either getting the syndie softsuits with can be placed in your bag or an engineer is giving you a spare suit/you break in to the Engineering outpost and steal the spare hardsuit there.

Sprite wise it's steam punk themed with lots of gears and goggles on it and keeping it slightly larger than normal in some areas to keep up with the bulky theme that is in steam punk. The colors are also straight from several Engineering suits in general with only a small exception being the variety of gears around it. After some 10ish hours working, there's very few changes I think I can make to it that wouldn't ruin the bulky steampunk look and not differentiate from the original human sprite of it.

**Comment by ProperPants** [comment]
I think the steampunk look, while a neat idea, clashes with the general aesthetic of this server.

**Comment by TravellingMerchant** [comment]
I'm all for the idea. Just the current sprite needs a bit of improvement before being added.

**Comment by taukausanake** [comment]
Yes please, Mechanic needs a space suit but not a hardsuit. Being able to toggle your helmet is a QoL that they don't strictly need. Soft suit is fine.
Also why the weld protection when they already have goggles? Those are toggle-able already, you don't need the protection on the helmet. Being Engineering isn't good enough reason. They're like the redheaded stepchild of Engineering. Or Hispanic. They're there but no one cares about them

**Comment by pinatacolada** [comment]
i like how the mechanic is sponsored by mcdonalds on their locker

**Comment by LightFire53** [comment]
Yeah, I'm not convinced on the suit sprite. Why not just a regular EVA or engineering hardsuit?

**Comment by scrubmcnoob** [comment]
A unique sprite is always welcomed but maybe make this a soft suit and a softer looking sprite to reflect it?

**Comment by RemieRichards** [comment]
McDonalds locker.

**Comment by granodd** [comment]
Like the idea, not really digging the suit though.

**Comment by Cactus2388** [comment]
I don't like the hardsuit helmet, but YES for a mechanics hardsuit. I hate having paperwork HOP's say "why do you need EVA?"

**Comment by KasparoVy** [comment]
lul forget eva and just fly pod out into space with full robolimbs, wear internals, firesuit/gasmask/firehelmet/blackgloves emergency autoinject yourself and find the derelict or rob mining

**Comment by AffectedArc07** [comment]
As a regular mechanic player, I have asked and asked so many times to get a suit. We need this

**Comment by SamHPurp** [comment]
@AffectedArc07 a suit maybe, but a HARD suit? I don't think so, they're a mechanic... a soft suit should do.

**Comment by Xantholne** [comment]
@SamHPurp I changed it to a soft suit like 10 hours ago nerd.

**Comment by Regen1** [comment]
Soft suit is fine, however the sprites are.. Not.

They look iffy, it's not steampunk, there's little detail that shows any steampunk correlation except the eyes.

This also does not really fit in on the station, the colorscheme is not something we have on any other suit, and while it does make it stand out more, i'd much rather have something that's more represented by Nanotrasen blue or something.

**Comment by Fox-McCloud** [comment]
If mechanic is to get a suit, we can just give them a regular EVA suit or an engineering hardsuit.

They really don't need their own unique sprite or suit.

**Comment by Regen1** [comment]
>They really don't need their own unique sprite or suit.

This is true, however I'd like for them to be distinguishable, give them just a bit more recognisability, like the other karma jobs.

**Comment by Fox-McCloud** [comment]
They have enough distinguishability in the tools they have access to and what they do; they don't need a special suit that does exactly the same thing as an engineering hardsuit or soft-suit just for the sake of it having different colors.

Sec pod pilot doesn't get his own special snowflake hardsuit; I don't see any reason mechanic should.

(Someone is invariably going to bring up the paramedic, which I think is another statistical outlier that's not worth having---incidentally its sprite is horrible and also doesn't fit with the rest of the soft suits).

**Comment by SamHPurp** [comment]
@Fox-McCloud meh, but Paramedic isn't a karma role.

**Comment by Kyep** [comment]
I agree that the Mechanic deserves a suit.

**Comment by uraniummeltdown** [comment]
Even though you made the stats softsuit, it still looks like a hardsuit and nothing like the other hardsuits.
Don't make it steampunk (which that is not), just a simple recolor of the EVA or syndie softsuits would be fine

**Comment by taukausanake** [comment]
Maybe a resprited orange/colored EVA suit, recolored to yellow so it's matching engineering. That should be unique enough

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -116,7 +116,19 @@
 	armor = list(melee = 10, bullet = 5, laser = 10, energy = 5, bomb = 10, bio = 100, rad = 0)
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 	flash_protect = 2
-	
+
+/obj/item/clothing/suit/space/eva/plasmaman/mechanic
+	name = "plasmaman mechanic suit"
```
Comment (taukausanake) [inline code/modules/clothing/spacesuits/plasmamen.dm]: Wait, aren't these guys already space worthy? Unless this is their actual uniform which didn't exist until now

Comment (Anoonki) [inline code/modules/clothing/spacesuits/plasmamen.dm]: They already were, but the suit they got as a mechanic was the plasmaman civilian spacesuit which was really silly. IIRC.

Comment (LightFire53) [inline code/modules/clothing/spacesuits/plasmamen.dm]: @xantholne do we have an image for the plasmaman?

**Comment by Aurorablade** [comment]
You didn't need to close this, :/

**Pull request closed without merging**